Terminal.Gui/Application.cs

```diff
@@ -1080,49 +1117,62 @@ static void OnUnGrabbedMouse (View view)
 			UnGrabbedMouse?.Invoke (view, new ViewEventArgs (view));
 		}
 
+		static View _lastMouseOwnerView;
+
 		/// <summary>
-		/// Merely a debugging aid to see the raw mouse events
+		/// Event fired when a mouse move or click occurs. Coordinates are screen relative.
 		/// </summary>
-		public static Action<MouseEvent> RootMouseEvent { get; set; }
-
-		static View _lastMouseOwnerView;
+		/// <remarks>
+		/// <para>
+		/// Use this event to receive mouse events in screen coordinates. Use <see cref="Responder.MouseEvent"/> to receive
+		/// mouse events relative to a <see cref="View"/>'s bounds.
+		/// </para>
+		/// <para>
+		/// The <see cref="MouseEvent.View"/> will contain the <see cref="View"/> that contains the mouse coordinates.
+		/// </para>
+		/// </remarks>
+		public static event EventHandler<MouseEventEventArgs> MouseEvent;
 
-		static void ProcessMouseEvent (MouseEvent me)
+		/// <summary>
+		/// Called when a mouse event occurs. Fires the <see cref="MouseEvent"/> event.
+		/// </summary>
+		/// <param name="a"></param>
+		public static void OnMouseEvent (MouseEventEventArgs a)
```

Comment:
I really don't understand why you are exposing as public. Why a user will call this and also the keys events? You are changing public objects to internal and now you are changing private objects to public. To call this method a user need to provide a `MouseEvent` and will send a key that wasn't returned from the terminal itself. This method is to propagate an event from the terminal to the focused or grabbed view.

Reply:
See the unit tests and how I updated them to use this. This is a far better API for being able to "simulate a mouse event". It could be argued it should be `internal`.

Reply:
Ak ok is for simulate then is all right, sorry.

Reply:
I just pushed a change that updated the API docs to reflect this.